### ISSUE TITLE
Find member by email adress

### ISSUE THREAD

**Issue by kazemar** [kalisio/kdk#442]
It could be usefull to find members by email : if I don't remember a member's name it can be difficult to find him. Besides, a member can change himself his name without notification for the organisation's manager.

**Comment by claustres** [comment]
Yes, it was previously the case but has been removed in the last refactoring.
